nnstreamer_example/custom_example_opencv/meson.build

```diff
@@ -1,17 +1,17 @@
-opencv_dep = dependency('opencv')
+opencv_dep = dependency('opencv', required: false)
```

Comment:
Will this work in the case when opencv is installed from source to a local location (without updating pkg-config)?
You can consider using ldconfig to verify is opencv is available (refer to #1336)

Reply:
Yes, the old script would break as well.